jsonschema/src/error.rs

```diff
@@ -927,14 +927,14 @@ impl fmt::Display for ValidationError<'_> {
             ),
             ValidationErrorKind::MaxProperties { limit } => write!(
                 f,
-                "{} has more than {} propert{}",
+                "{} has more than {} property{}",
```

Comment:
This one is conditional, see the last argument to the macro

Reply:
reverted that